nightingale_dispatcher/src/nightingale_dispatcher/mission_planner.py

```diff
@@ -100,16 +100,21 @@ def get_items_phase(self):
         # Arrived at stock area
```

Comment:
should these changes be part of this PR? I feel like these should go with manipulation changes